apps/dialtone-documentation/docs/components/toggle.md

```diff
@@ -64,60 +64,46 @@ The Toggle component acts as a way to allow the User to switch between two mutua
-        <span class="d-toggle__inner" />
-      </button>
-    </div>
-  </div>
-  <div class="d-toggle-group d-d-flex d-ai-center">
-    <div class="d-toggle-copy">
-      <label class="d-toggle-label" for="Dialtone-Toggle2">Checked Toggle</label>
-    </div>
-    <div class="d-toggle-button">
-      <button class="d-toggle d-toggle--checked" type="button" role="switch" id="Dialtone-Toggle2" >
-        <span class="d-toggle__inner" />
-      </button>
-    </div>
-  </div>
-  <div class="d-toggle-group d-d-flex d-ai-center">
-    <div class="d-toggle-copy">
-      <label class="d-toggle-label" for="Dialtone-Toggle3">Unchecked Disabled</label>
-    </div>
-    <div class="d-toggle-button">
-      <button class="d-toggle" type="button" role="switch" disabled id="Dialtone-Toggle3" >
-        <span class="d-toggle__inner" />
-      </button>
-    </div>
-  </div>
-  <div class="d-toggle-group d-d-flex d-ai-center">
-    <div class="d-toggle-copy">
-      <label class="d-toggle-label" for="Dialtone-Toggle4">Checked Disabled</label>
-    </div>
-    <div class="d-toggle-button">
-      <button class="d-toggle d-toggle--checked" type="button" role="switch" disabled id="Dialtone-Toggle4" >
-        <span class="d-toggle__inner" />
-      </button>
-    </div>
-  </div>
-  <div class="d-toggle-group d-d-flex d-ai-center">
-    <div class="d-toggle-copy">
-      <label class="d-toggle-label" for="Dialtone-Toggle5">Indeterminate Toggle</label>
-    </div>
-    <div class="d-toggle-button">
-      <button class="d-toggle d-toggle--indeterminate" type="button" role="checkbox" aria-checked="mixed" id="Dialtone-Toggle5">
-        <span class="d-toggle__inner" />
-      </button>
-    </div>
-  </div>
-</fieldset>
-```
+<code-example-tabs
+htmlCode='
+<div class="d-toggle-wrapper">
+  <label for="dt3"> Unchecked Toggle </label><button id="dt3" role="switch" type="button" aria-checked="false" aria-disabled="false" class="d-toggle"><span class="d-toggle__inner"></span></button>
+</div>
+<div class="d-toggle-wrapper">
+  <label for="dt4"> Checked Toggle </label>
+  <button id="dt4" role="switch" type="button" aria-checked="true" aria-disabled="false" class="d-toggle d-toggle--checked"><span class="d-toggle__inner"></span></button>
+</div>
+<div class="d-toggle-wrapper">
+  <label for="dt5"> Unchecked Disabled </label>
+  <button id="dt5" role="switch" type="button" aria-checked="false" aria-disabled="true" class="d-toggle d-toggle--disabled" disabled="disabled"><span class="d-toggle__inner"></span></button>
+</div>
+<div class="d-toggle-wrapper">
+  <label for="dt6"> Checked Disabled </label>
+  <button id="dt6" role="switch" type="button" aria-checked="true" disabled="disabled" aria-disabled="true" class="d-toggle d-toggle--checked d-toggle--disabled"><span class="d-toggle__inner"></span></button>
+</div>
+<div class="d-toggle-wrapper">
+  <label for="dt7"> Indeterminate Disabled </label>
```

Comment:
The live example is not disabled, yet it is disabled in this code example

Reply:
Right, updated the live example 👍🏼